cmd/gosky/main.go

```diff
@@ -765,7 +765,7 @@ var listLabelsCmd = &cli.Command{
 		since := time.Now().Add(-1 * delta).UnixMilli()
 
 		xrpcc := &xrpc.Client{
-			Host: "https://api.bsky.app",
+			Host: "https://mod.bsky.app",
```

Comment:
would be good to have this be a config (env var) as well, but not a blocking issue